src/js/outreach/views/opt-in.scss

```diff
@@ -21,6 +21,29 @@
   width: 100%;
 }
 
+.opt-in__field-select {
```

Comment:
these styles can be removed.. they're down in the select module which looks good to me

Reply:
Ooops, missed that. That's been removed now 👍